dashboard/app/controllers/level_assets_controller.rb

```diff
@@ -5,10 +5,10 @@ class LevelAssetsController < ApplicationController
   def upload
     authorize! :create, Level
 
-    # Only allow images that are <= 1MB
-    if params[:file].size > 1.megabyte
+    # Only allow images that are <= 2MB
+    if params[:file].size > 2.megabyte
       return render json: {
-        message: "asset too large; images must be no larger than 1MB",
+        message: "asset too large; images must be no larger than 2MB",
```

Comment:
changes to this file look good!

pdftk changes in this PR look like they do not belong here.

also, there is this line which I think we might want to update too: https://github.com/code-dot-org/code-dot-org/blob/7c640fd23d498475a5b4e9ea7c2d186864314ec6/apps/src/code-studio/initializeCodeMirror.js#L138 up to you whether to try and figure out when one error message gets used or the other -- I would lean toward changing both and then waiting to see if there are issues.

Reply:
ooh thanks - I'm not sure how those pdftk commits got included, but will remove!